### ISSUE TITLE
Untaught Type Error:  when launching template for daily notes

### ISSUE THREAD

**Issue by moorebi** [SilentVoid13/Templater#506]
I am getting an error when I launch my daily notes template… the console is giving me this information:

‘ Uncaught TypeError: Cannot read property 'bind' of undefined at eval (eval at <anonymous> (app.js:1), <anonymous>:7083:109)
quotes.rest/qod:1 Failed to load resource: net::ERR_CERT_AUTHORITY_INVALID
Log.ts:17 Templater Error: Template parsing error, aborting. 
 Failed to fetch
log_error @ Log.ts:17

2VM147:7083 Uncaught TypeError: Cannot read property 'bind' of undefined
    at eval (eval at <anonymous> (app.js:1), <anonymous>:7083:109)
quotes.rest/qod:1 Failed to load resource: net::ERR_CERT_AUTHORITY_INVALID
Log.ts:17 Templater Error: Template parsing error, aborting. 
 Failed to fetch
log_error @ Log.ts:17’

**Plugin informations (please complete the following information):**

-   OS: [Windows 10]
-   Templater version: [1.99]
-   Obsidian version: [0.13.19]
-   Templater settings: [bin/templates]

**Describe the bug**

THis is a snippet of the template that is having the issue:

---
tags: daily
aliases: 
date: <%tp.date.now("YYYY-MM-DD")%>
cssclass: 
---
Links: [[010 day planner moc]] | 📆 [[<%tp.date.now("YYYY-MM-DD", -1)%>]] <--> [[<%tp.date.now("YYYY-MM-DD", 1)%>]] |

## Daily Quote
> <% tp.web.daily_quote() %>

---
## Day Planner

**Expected behavior**

**Screenshots**

**Additional context**

**Comment by shabegom** [comment]
@moorebi There is an issue with the daily quotes command right now. Remove that command and your template should start working again. 

You can use this command for quotes as a workaround:

```
<%*
const response = await fetch(`https://api.quotable.io/random`);
const     qod = await response.json();
const finalQuote = `> ${qod.content}
>
> &mdash; <cite>${qod.author}</cite>✍️`
tR += finalQuote
%>
```

**Comment by zhqqqy** [comment]
Daily Quote will access https://quotes.rest/qod websits ,But this site's certificate has expired on January 11 in 2021。

**Comment by shabegom** [comment]
Resolved in 1.9.10
